tests/src/test/java/alluxio/client/fs/DirectAccessIntegrationTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.fs;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileInStream;
+import alluxio.client.file.FileOutStream;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.FileSystemUtils;
+import alluxio.conf.PropertyKey;
+import alluxio.grpc.CreateFilePOptions;
+import alluxio.testutils.LocalAlluxioClusterResource;
+
+import com.google.common.io.ByteStreams;
+import com.google.common.io.Files;
+import org.apache.commons.io.IOUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.File;
+
+/**
+ * Integration tests for path level configurations.
+ */
+public class DirectAccessIntegrationTest {
+  private static final byte[] TEST_BYTES = "TestBytes".getBytes();
+  private static final int USER_QUOTA_UNIT_BYTES = 1000;
+  private static final String DIRECT_DIR = "/mnt/direct/";
+  private static final String NON_DIRECT_DIR = "/mnt/non_direct/";
+
+  @Rule
+  public LocalAlluxioClusterResource mLocalAlluxioClusterResource =
+      new LocalAlluxioClusterResource.Builder()
+          .setProperty(PropertyKey.USER_FILE_BUFFER_BYTES, USER_QUOTA_UNIT_BYTES)
+          .setProperty(PropertyKey.USER_FILE_DIRECT_ACCESS, DIRECT_DIR)
+          .build();
+  private FileSystem mFileSystem;
+
+  private final String mLocalUfsPath = Files.createTempDir().getAbsolutePath();
```

Comment:
Use `AlluxioTestDirectory` which automatically cleans up the directory after the test is finished. Take this as an example:

https://github.com/Alluxio/alluxio/blob/6dc8ed2391cce156418cf6c6cce3f44cbe62e99b/dora/core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTestBase.java#L154-L155